integration-tests/README.md

```diff
@@ -31,18 +31,50 @@ docker info
 Running Integration tests
 =========================
 
-## Running tests using mvn
+## Starting docker tests
 
-To run all the tests using mvn run the following command -
+To run all the tests using docker and mvn run the following command -
 '''''
   mvn verify -P integration-tests
 '''''
 
-To run only a single test using mvn run following command -
+To run only a single test using mvn run the following command -
 '''''
   mvn verify -P integration-tests -Dit.test=<test_name>
 '''''
 
+## Configure and run integration tests using existing cluster
+
+To run tests on any druid cluster that is already running, create a configuration file:
+
+    {   
+       "broker_host": "<broker_ip>",
+       "broker_port": "<broker_port>",
+       "router_host": "<router_ip>",
+       "router_port": "<router_port>",
+       "indexer_host": "<indexer_ip>",
+       "indexer_port": "<indexer_port>",
+       "coordinator_host": "<coordinator_ip>",
+       "coordinator_port": "<coordinator_port>",
+       "middle_manager_host": "<middle_manager_ip>",
+       "zookeeper_host": "<zookeeper_ip>",
```

Comment:
Issue #1523 exists for an option to use service discovery to get node address using a zookeeper location.

Currently we do not need more than one IP for zookeeper, but it would be easy enough to allow a comma-separated string as the value for the zookeeper address.